docs/content/tutorials/ingestion.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc_page
+---
+
+# Loading Data
+
+## Choosing an ingestion method
+
+Druid supports streaming (real-time) and file-based (batch) ingestion methods. The most
+popular configurations are:
+
+- [Files](batch-ingestion.html) - Load data from HDFS, S3, local files, or any supported Hadoop
+filesystem in batches. We recommend this method if your dataset is already in flat files.
+
+- [Stream push](stream-ingestion.html#stream-push) - Push a data stream into Druid in real-time
+using [Tranquility](http://github.com/druid-io/tranquility), a client library for sending streams
+to Druid. We recommend this method if your dataset originates in a streaming system like Kafka,
+Storm, Spark Streaming, or your own system.
+
+- [Stream pull](stream-ingestion.html#stream-pull) - Pull a data stream directly from an external
+data source into Druid using Realtime Nodes.
+
+## Getting started
+
+The easiest ways to get started with loading your own data are the three included tutorials.
+
+- [Files-based tutorial](tutorial-batch.html) showing you how to load files from your local disk.
+- [Streams-based tutorial](tutorial-streams.html) showing you how to push data over HTTP.
+- [Kafka-based tutorial](tutorial-kafka.html) showing you how to load data from Kafka.
+
+## Hybrid batch/streaming
+
+You can combine batch and streaming methods in a hybrid batch/streaming architecture. In a hybrid
+architecture, you use a streaming method to do initial ingestion, and then periodically re-ingest
+older data in batch mode (typically every few hours, or nightly).
+
+Hybrid architectures are simple with Druid, since batch loaded data for a particular time range
+automatically replaces streaming loaded data for that same time range. All Druid queries seamlessly
```

Comment:
it can be hard to describe the  hybrid approach without good docs on how the segment timeline works... I should add that in another PR